test/Vagrantfile

```diff
@@ -6,8 +6,8 @@ $JOB_NAME = ENV['JOB_BASE_NAME'] || "LOCAL"
 $K8S_VERSION = ENV['K8S_VERSION'] || "1.8"
 $K8S_NODES = (ENV['K8S_NODES'] || "2").to_i
 
-$SERVER_BOX= "cilium/ginkgo"
-$SERVER_VERSION="0.0.4"
+$SERVER_BOX= "eloycoto/ubuntu_test"
```

Comment:
The images we provide should not be in a private user's account, only under `cilium/`.

Reply:
Discussed offline, this is due to issues with VagrantCloud uploading. We will use this as a stopgap until we get better storage resources for the official image.